### ISSUE TITLE
Hi, May I suggestions one thing?

### ISSUE THREAD

**Issue by MurlocTinyfin** [HearthSim/Hearthstone-Deck-Tracker#2819]
Hi. nice to meet you.
May I suggestions one thing?

When the 2card is draw in hand and when there is no card both have the same color(Gray).
Only when no card in hand and deck change the color gray in the Deck Windows.

If card is in the hand after 2card draw from My Deck, did not change the color gray in the Deck Windows. 
When i confused the 2card is draw in hand and when there is no card like the Gray color.
It seems nothing, but I'm confused.

When the draw 2card, if i have a card is do not changed to gray color in the Deck Windows.
Only when no card in hand and deck change the color gray in the Deck Windows.

Is it possible to do this? Please.

Finally,
Thank you. Thank you for making a good program.

So good bye.

p.s : I do not speak English well. I used Google Translator. Hope you understand. Sorry.

**Comment by Shyran** [comment]
What you are saying is, if you have 2 of a card and you've played one, and the other one is in your hand, do not make that card the darker gray in the deck list until after you've played the second one. Is this correct?

**Comment by MurlocTinyfin** [comment]
Right. After using one card, when the card is in the hand other one, DeckWindows is changed gray. In addition, there are two cards in hand, the DeckWindows turns gray. As if no card. Although the card is in hand, it feels like it does not have the card. confused. (Do you understand? I'm sorry I can not speak English well)

**Comment by riQQ** [comment]
Do you have `Highlight cards in hand enabled`? Then cards in your hand should be marked green to indicate you have at least one copy in your hand and if there are no more copies in your deck, the card should be greyed out in addition.

**Comment by Shyran** [comment]
> Do you have Highlight cards in hand enabled (under Options > Overlay > Player)? Then cards in your hand should be marked green to indicate you have at least one copy in your hand and if there are no more copies in your deck, the card should be greyed out in addition.

Even so, what they are requesting is that the cards in their deck list not get the dark gray color until after both cards have actually been played. I understand the concept, if the two copies of a card are both in my hand, then technically they are no longer in my deck, and since its a deck list, they get grayed out,
